contracts/modules/TransferManager/ManualApprovalTransferManager.sol

```diff
@@ -19,20 +19,19 @@ contract ManualApprovalTransferManager is ITransferManager {
 
     //Manual approval is an allowance (that has been approved) with an expiry time
     struct ManualApproval {
+        address from;
+        address to;
         uint256 allowance;
         uint256 expiryTime;
-    }
-
-    //Manual blocking allows you to specify a list of blocked address pairs with an associated expiry time for the block
-    struct ManualBlocking {
-        uint256 expiryTime;
+        bytes32 description;
     }
 
     //Store mappings of address => address with ManualApprovals
     mapping (address => mapping (address => ManualApproval)) public manualApprovals;
```

Comment:
Instead of maintaining this redundant struct mapping, can we just use `approvalsIndex` mapping to fetch index and then fetch required data from `activeManualApprovals` using that index?